sdk/communication/azure-communication-rooms/README.md

```diff
@@ -0,0 +1,228 @@
+
+```java readme-sample-deleteRoomWithRoomId
+public void deleteRoomWithRoomId() {
+    RoomsClient roomsClient = createRoomsClientWithConnectionString();
+    try {
+        roomsClient.deleteRoomWithResponse("<Room Id in String>", Context.NONE);
+    } catch (RuntimeException ex) {
+        System.out.println(ex);
+    }
+}
+```
+
+### Add participants an existing room
+Use the `addParticipants`  function to add participants to an existing Room on Azure Communciation Service.
+
+```java readme-sample-addRoomParticipantsWithRoomId
+public void addRoomParticipantsWithRoomId() {
+    RoomParticipant user1 = new RoomParticipant().setCommunicationIdentifier(new CommunicationUserIdentifier("8:acs:b6372803-0c35-4ec0-833b-c19b798cef2d_0000000e-3240-55cf-9806-113a0d001dd9")).setRole(RoleType.ATTENDEE);
+    RoomParticipant user2 = new RoomParticipant().setCommunicationIdentifier(new CommunicationUserIdentifier("8:acs:b6372803-0c35-4ec0-833b-c19b798cef2d_0000000e-3240-55cf-9806-113a0d001dd7")).setRole(RoleType.PRESENTER);
+    RoomParticipant user3 = new RoomParticipant().setCommunicationIdentifier(new CommunicationUserIdentifier("8:acs:b6372803-0c35-4ec0-833b-c19b798cef2d_0000000e-3240-55cf-9806-113a0d001dd5")).setRole(RoleType.CONSUMER);
+
+    List<RoomParticipant> participants = new ArrayList<RoomParticipant>(Arrays.asList(user1, user2, user3));
+    RoomsClient roomsClient = createRoomsClientWithConnectionString();
+
+    try {
+        ParticipantsCollection roomParticipants =  roomsClient.addParticipants("<Room Id>", participants);
+        System.out.println("No. of Participants in Room: " + roomParticipants.getParticipants().size());
+
+    } catch (RuntimeException ex) {
+        System.out.println(ex);
+    }
+}
+```
+
+### Remove participants an existing room
+Use the `removeParticipants`  function to remove participants from an existing Room on Azure Communciation Service.
+
+```java readme-sample-removeRoomParticipantsWithRoomId
+public void removeRoomParticipantsWithRoomId() {
+    RoomParticipant user1 = new RoomParticipant().setCommunicationIdentifier(new CommunicationUserIdentifier("8:acs:b6372803-0c35-4ec0-833b-c19b798cef2d_0000000e-3240-55cf-9806-113a0d001dd9")).setRole(RoleType.ATTENDEE);
+    RoomParticipant user2 = new RoomParticipant().setCommunicationIdentifier(new CommunicationUserIdentifier("8:acs:b6372803-0c35-4ec0-833b-c19b798cef2d_0000000e-3240-55cf-9806-113a0d001dd7")).setRole(RoleType.PRESENTER);
+
+    List<RoomParticipant> participants = new ArrayList<RoomParticipant>(Arrays.asList(user1, user2));
+    RoomsClient roomsClient = createRoomsClientWithConnectionString();
+
+    try {
+        ParticipantsCollection roomParticipants =  roomsClient.removeParticipants("<Room Id>", participants);
+        System.out.println("Room Id: " + roomParticipants.getParticipants().size());
+
+    } catch (RuntimeException ex) {
+        System.out.println(ex);
+    }
+}
+```
+
+## Troubleshooting
+
+In progress.
```

Comment:
We should include the basic troubleshooting instructions here - like enabling logging.

Reply:
Added troubleshooting steps.